tests/unit-tests/dds/security/AccessControlTest.cpp

```diff
@@ -26,8 +26,8 @@ using namespace testing;
 static const char* Expected_Permissions_Token_Class_Id ="DDS:Access:Permissions:1.0";
```

Comment:
The line for this unit test is still the same in [`tests/security/security_tests.lst`](https://github.com/objectcomputing/OpenDDS/blob/7561732ba6d9edfa81485460962714c7ced29a4e/tests/security/security_tests.lst#L3).